### DISCUSSION TITLE
Include app.css in custom error.html file

### DISCUSSION THREAD

**Discussion by kevinrenskers** [sveltejs/kit#6801] [Q&A] [answered]
I have an `app.css` file in my `/src` folder, which I simply import in my root layout `import "../app.css"`. Now I want to create my own custom `error.html` file, and I want to include the same base `app.css` file, but there is no static URL for it that I can import. If I move `app.css` to the `/static` folder I can easily include it from the custom `error.html` file, but then I can't import it anymore from my root layout. So then I'd have to include it in the `app.html` header myself and it won't get bundled, I don't get HMR when making changes to the file, etc.

So other than duplicating this file, what other options are there for sharing the base css between the site and the static error page?

**Comment by dan-livingston** [comment, marked as answer]
One option is to create a route, say `routes/app.css/+server.js`, that you use to serve the css file.

**Reply by kevinrenskers** [reply]
Ah, nice one
